bundles/best_practices.rst

```diff
@@ -166,6 +166,69 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+    env:
+        global:
+            - TEST_COMMAND="phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported PHP and Symfony version
+            - php: 5.5
+              env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="phpunit --coverage-text"
+
+              # Test the latest stable release
+            - php: 5.5
+            - php: 5.6
+            - php: 7.0
+            - php: 7.1
+            - php: 7.2
+
+              # Test LTS versions
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:v2"
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:v3"
+
+              # Latest beta release
+            - php: 7.2
+              env: DEPENDENCIES="beta"
```

Comment:
people should rather test latest dev than latest beta. Most libraries don't release beta releases (Twig for instance will not have a beta release before the stable one). The goal of this job is to catch incompatibilities with future releases as early as possible.
This job is not only about Symfony itself, but also about other libraries they depend on.